calcom/cal.com
# fix: Wrong text for team admin that is not part of booking

## What does this PR do?
The PR is about resolving issue #16392

If admin/owner views booking for which they are not part of it, then "You are meeting with" is visible. Changed it from "You are in meeting" to "attendees is meeting with user "

![image](https://github.com/user-attachments/assets/1e938c90-9fcf-452d-9cd4-55ef5d9b451f)
  
- Fixes #16392



## Mandatory Tasks (DO NOT REMOVE)

- [N/A] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.

## How should this be tested?



- Create a round robin event type for 2 members of a team.
- Visit the booking page and go to the above booking details as a admin/owner.
- Notice that admin/owner sees 'is meeting with' instead of ' You are meeting with' 
## Checklist
